PayPalMessaging/src/test/java/com/braintreepayments/api/PayPalMessagingColorUnitTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.braintreepayments.api
+
+import com.paypal.messages.config.message.style.PayPalMessageColor
+import junit.framework.TestCase.assertEquals
+import org.junit.Test
+
+class PayPalMessagingColorUnitTest {
+
+    @Test
+    fun testColor_withBlack_returnsRawValueBlack() {
```

Comment:
Extreme nitpick but do we want to move towards the Kotlin backtick naming convention for Kotlin tests? We've started doing that in v5 but don't have consistency across the board. So like `test black color returns raw value black` (surrounded in backticks)

Reply:
Done: https://github.com/braintree/braintree_android/pull/925/commits/e61e1b1fcbaa70faacba3255fa37fabc2516ecc9

Reply:
This was a suggestion from @tdchow in another PR - we are still working on determining our Kotlin style guide, but the back tick test names just make it easier to read test failures when run